src/pages/Inventory.page.js

```diff
@@ -9,6 +9,23 @@ export class InventoryPage extends BaseSwagLabPage {
 
     get addItemToCartBtns() { return this.page.locator('[id^="add-to-cart"]'); }
 
+    get inventoryItemName() { return this.page.locator('.inventory_item_name '); }
+
+    get inventoryItemPrice() { return this.page.locator('.inventory_item_price'); }
+
+    async selectSortOption(value) {
+        await this.productSorting.selectOption(value);
+    }
+
+    async verifySortingNames() {
+        return this.inventoryItemName.allTextContents();
+    }
+
+    async verifySortingPrices() {
```

Comment:
is this function verifying smth? better to name it getPricesValues or smth like that